PRMSU-FACULTY-MANAGEMENT-SYSTEM/src/AddFaculty.java

```diff
@@ -0,0 +1,197 @@
+        middleInitial.setColumns(10);
+        middleInitial.setBorder(new TitledBorder(new LineBorder(new Color(192, 192, 192), 2, true), "MIDDLE INITIAL", TitledBorder.LEFT, TitledBorder.TOP, null, new Color(0, 0, 0)));
+        middleInitial.setBackground(SystemColor.menu);
+        middleInitial.setBounds(30, 180, 440, 50);
+        addFacultyFrame.getContentPane().add(middleInitial);
+        
+        extName = new JTextField();
+        extName.setFont(new Font("Rockwell", Font.BOLD, 20));
+        extName.setDisabledTextColor(Color.BLACK);
+        extName.setColumns(10);
+        extName.setBorder(new TitledBorder(new LineBorder(new Color(192, 192, 192), 2, true), "EXTENSION NAME", TitledBorder.LEFT, TitledBorder.TOP, null, new Color(0, 0, 0)));
+        extName.setBackground(SystemColor.menu);
+        extName.setBounds(30, 240, 440, 50);
+        addFacultyFrame.getContentPane().add(extName);
+        
+        JButton addFacultyBtn = new JButton("Add");
+        // ** WAITING **
+//        addFacultyBtn.addActionListener(new ActionListener() {
+//        	public void actionPerformed(ActionEvent e) {
+//        	}
+//        });
+        addFacultyBtn.setBorder(new MatteBorder(5, 3, 5, 3, (Color) new Color(192, 192, 192)));
+        addFacultyBtn.setBackground(new Color(240, 240, 240));
+        addFacultyBtn.setFont(new Font("Elephant", Font.PLAIN, 20));
+        addFacultyBtn.setBounds(150, 475, 200, 50);
+        addFacultyFrame.getContentPane().add(addFacultyBtn);
+        
+        JButton cancelBtn = new JButton("Cancel");
+        cancelBtn.addActionListener(new ActionListener() {
+        	public void actionPerformed(ActionEvent e) {
+        		if (e.getSource() == cancelBtn) {
+        			addFacultyFrame.hide();
+        		}
+        		
+        	}
+        });
+        cancelBtn.setFont(new Font("Elephant", Font.PLAIN, 20));
+        cancelBtn.setBorder(new MatteBorder(5, 3, 5, 3, (Color) new Color(192, 192, 192)));
+        cancelBtn.setBackground(new Color(240, 240, 240));
+        cancelBtn.setBounds(150, 535, 200, 50);
+        addFacultyFrame.getContentPane().add(cancelBtn);
+        
+        JComboBox departmentCBox = new JComboBox();
+        departmentCBox.setBackground(new Color(240, 240, 240));
+        departmentCBox.setName("DEPARTMENT");
+        departmentCBox.setBounds(30, 300, 440, 40);
+        addFacultyFrame.getContentPane().add(departmentCBox);
+        
+        acadYearCBox = new JComboBox();
+        acadYearCBox.setName("DEPARTMENT");
+        acadYearCBox.setBounds(30, 350, 440, 40);
+        addFacultyFrame.getContentPane().add(acadYearCBox);
+        
+        semesterCBox = new JComboBox();
+        semesterCBox.setName("DEPARTMENT");
+        semesterCBox.setBounds(30, 400, 440, 40);
+        addFacultyFrame.getContentPane().add(semesterCBox);
+        
+        
+        //Combo Box Items
+        
+        departmentCBox.addItem("DEPARTMENT");

```

Comment:
Sa Combo Box ng department dapat nakalagay lang dun ay Civil Engineering, Mechanical Engineering, Electrical Engineering, at Computer Engineering

Reply:
and Allied